types/tektelic/smartRoomPIR/occupancy.schema.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$id": "https://akenza.io/tektelic/smartRoomSensorPIR/occupancy.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Occupancy",
+  "type": "object",
+  "properties": {
+    "motion": {
+      "title": "Motion",
+      "description": "The number of motion counts.",
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 255
+    },
+    "occupancy": {
```

Comment:
please align this with other occupancy measurements where we use boolean or not?

Reply:
We usually use integers

Reply:
This is not true...
https://github.com/akenza-io/device-type-library/blob/master/types/decentlab/IAM/occupancy.schema.json
https://github.com/akenza-io/device-type-library/blob/master/types/internal/deviceSimulator/default.schema.json

please align to have a unified format